_data/authors.yml

```
@@ -259,8 +259,8 @@ parkera:
    about: "Tony Parker manages teams at Apple working on Foundation and Swift packages."
 
 diversity-work-group:
```

```suggestion
diversity-workgroup:
```
We should also update the blog post which contains this reference

---

- _posts/2020-12-16-accessibility-and-inclusion.md
- _posts/2020-12-16-diversity-in-swift.md
- _posts/2021-05-10-swift-mentorship-program.md